test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestIRMatching.java

```diff
@@ -1486,7 +1486,7 @@ public void macro2() {
 
     @Test
     @IR(failOn = IRNode.ALLOC)
-    @IR(counts = {IRNode.COUNTED_LOOP, "1"}) // not fail
+    @IR(counts = {IRNode.COUNTED_LOOP, ">1"}) // not fail
```

Comment:
Can you explain what led to the difference? Can you also set an upper bound?

Reply:
Is this going to be GC independent? What if the VM swaps to another GC ergonomically?

Reply:
I suppose that's probably not possible, as far as I know we always run with G1GC, so it should be fine :)
We could put an upper bound of 3, maybe 4 here though. Just see what passes.